code/modules/vending/_vending.dm

```diff
@@ -723,6 +736,12 @@ GLOBAL_LIST_EMPTY(vending_products)
 			ref = REF(R),
 			premium = TRUE
 		)
+		if(SSevents.holidays && SSevents.holidays[PRIDE_WEEK])
+			data["name"] = "pride [R.name]"
+			if(R.custom_price)
+				data["price"] = R.custom_premium_price * 1.25
+			else
+				data["price"] = extra_price * 1.25
```

Comment:
```suggestion
			data["price"] = 1.25 * R.custom_price ? R.custom_premium_price : extra_price
```

Reply:
I have to manually merge all of these in because skoglol felt the need to violate the 24 hour rule, but thanks for putting in the effort to do these as suggestions.